config-windows.md

```diff
@@ -19,6 +19,32 @@ The Windows container specification uses APIs provided by the Windows Host Compu
     }
 ```
 
+## <a name="configWindowsDevices" />Devices
+
+**`devices`** (array of objects, OPTIONAL) lists devices that MUST be available in the container.
+
+Each entry has the following structure:
+
+* **`id`** *(string, REQUIRED)* - specifies the device which the runtime MUST make available in the container.
+* **`id_type`** *(string, REQUIRED)* - tells the runtime how to interpret `id`. Today, Windows only supports a value of `class`, which identifies `id` as a [device interface class GUID][interfaceGUID].
+
+[interfaceGUID]: https://docs.microsoft.com/en-us/windows-hardware/drivers/install/overview-of-device-interface-classes
+
+### Example
+
+```json
+    "devices": [
```

Comment:
nit: The other examples in `config-windows.md` include the `"windows"` object to which this is an entry.  What you have here, on the other hand, is closer to the example approach taken in `config-linux.md`, where you only show the key being defined in this subsection.  I don't know if the maintainers have a preference, but personally I think consistency within this file is more important.  And ideally we'd have consistency across the whole spec, so we wouldn't have to make these decisions for each new example ;).